### PR TITLE
swap fixes

### PR DESCRIPTION
**Pull request by vineet-webonise**

### CODE REVIEW

File: SUNFLOWR/SUNFLOWR/MindVC/MindVC.swift [hunk 1]
```diff
@@ -400,7 +400,7 @@ extension MindVC: AddSwapDelegate {
             editActionVM.dictEditActionValues = dictEditData
             editActionVM.addContentApiCall(view: self.view)
         case ActivityName.Swap:
-            extSubSectionId = self.editActionVM.dictEditActionValues[Constants.EditData.NEW_SECTION_ID] as? String ?? ""
```
Comment (saketPandhare-Webonise) [inline SUNFLOWR/SUNFLOWR/MindVC/MindVC.swift:403]: avoid force unwrap 
self.addSwapPopOver.selectedSubSectionId as? String ?? ""

File: SUNFLOWR/SUNFLOWR/MoveVC/MoveVC.swift [hunk 2]
```diff
@@ -352,7 +352,7 @@ extension MoveVC: AddSwapDelegate {
             editActionVM.dictEditActionValues = dictEditData
             editActionVM.addContentApiCall(view: self.view)
         case ActivityName.Swap:
-            extSubSectionId = self.editActionVM.dictEditActionValues[Constants.EditData.NEW_SECTION_ID] as? String ?? ""
```
Comment (saketPandhare-Webonise) [inline SUNFLOWR/SUNFLOWR/MoveVC/MoveVC.swift:355]: same here

File: SUNFLOWR/SUNFLOWR/MindVC/MindVC.swift (hunk 1 above)
Comment (vineet-webonise) [reply SUNFLOWR/SUNFLOWR/MindVC/MindVC.swift:403]: Done

File: SUNFLOWR/SUNFLOWR/MoveVC/MoveVC.swift (hunk 2 above)
Comment (vineet-webonise) [reply SUNFLOWR/SUNFLOWR/MoveVC/MoveVC.swift:355]: Done

**Pull request merged**